src/components/siteTitle/index.scss

```diff
@@ -2,32 +2,45 @@
 
 .site-title {
   display: flex;
-  flex-direction: column;
+  flex-direction: row;
   align-items: center;
-  padding: 1rem;
+  padding: 0.5rem;
   margin: -1rem;
   width: min-content;
 }
 
 .site-title__logo {
-  height: 5rem;
+  height: 2rem;
+}
+
+.site-title:not(.site-title--vertical) {
+  & .site-title__name {
+    margin-left: 0.5rem;
+  }
 }
 
 .site-title__name-part {
   color: var(--text-pale);
-  font-size: 2rem;
+  font-size: 1rem;
   text-decoration: none;
   margin: 0;
   &:first-of-type {
     color: var(--text);
   }
 }
 
+@include md {
+  .site-title {
+    padding: 1rem;
+  }
+}
+
 @include lg {
-  .site-title:not(.site-title--vertical) {
-    flex-direction: row;
-    & .site-title__name {
-      margin-left: 0.5rem;
-    }
+  .site-title__name-part {
+    font-size: 2rem;
+  }
+
+  .site-title__logo {
+    height: 5rem;
   }
 }
```

Comment:
Just a general css comment (non-blocking) here:

Something cool that could have been done is using `em`, something that I almost never do.

The height of the logo and the font-size of the text have a close relationship. To ensure they always have the same ratio, we could set a font-size on the `.site-title` container, and make the logo use `em`. That would make the logo size relative to the font-size that we chose.

We could then just play with the font-size of `.site-title` on desktop/mobile to resize the whole logo.

If you're down for it, I am too. If not, no worries, it's basically a nerd trip because it would have literally 0 incident on the look of the website 🌞

Reply:
Let me try it

Reply:
Done!